service/templates/base_angular.html

```diff
@@ -6,10 +6,13 @@
-          ;NREUM.loader_config={accountID:"1728883",trustKey:"1728883",agentID:"447802388",licenseKey:"68c435e77c",applicationID:"447802388"}
-          ;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"68c435e77c",applicationID:"447802388",sa:1}
+          {{if .NewRelic}}
```

Comment:
This check won't work in go like it does in javascript. To check if the settings are set, you can either make the new relic settings a pointer in ui.go, or check that a specific field is not the blank string

Reply:
It's done [here](updateBookmarksOnChange) as well as a couple other places

Reply:
now i see what you mean and changed it